### DISCUSSION TITLE
How to display the number of screen frames

### DISCUSSION THREAD

**Discussion by WOLFxxxxxx** [bjornbytes/lovr#786] [General]
![maxresdefault](https://github.com/bjornbytes/lovr/assets/128394802/a15f0e3b-272a-434b-b96c-cc2e25b9dea4)

**Comment by bjornbytes** [comment]
`lovr.timer.getFPS` will return the FPS as a number.  From there you can log it with `print` or draw it to the screen with `Pass:text`.
